app/models/edm/provided_cho.rb

```diff
@@ -75,9 +71,9 @@ def edm_type_enum
 
     validates :dc_language, inclusion: { in: dc_language_enum.map(&:last) }, allow_blank: true
     validates :edm_type, inclusion: { in: edm_type_enum }, presence: true, if: :published?
-    validates_associated :dc_contributor_agent, :dc_subject_agents, :dcterms_spatial_places
+    validates_associated :dc_contributor_agent, :dc_subject_agents
     validates_with PresenceOfAnyValidator,
-                   of: %i(dc_subject dc_subject_agents dc_type dcterms_spatial dcterms_spatial_places dcterms_temporal),
+                   of: %i(dc_subject dc_subject_agents dc_type dcterms_spatial dcterms_temporal),
```

Comment:
this now results in: "one of DC subject, DC subject agents, DC type, DCTERMS spatial or DCTERMS temporal is required" which doesn't make sense to users because they won't be able to add 'DC type' or 'DCTERMS temporal' through the form.
Also for events staff are they expected to fill this data in through the cms? because they won't have fields on the 'edit' form for this either.

Reply:
Yes, this is not at all user-friendly, but it will be resolved elsewhere.

In order to differentiate migration campaign contributions from others, when provider and data provider are always "Europeana Foundation", we will add to every migration campaign contribution a dc:subject referencing a relevant concept, e.g. http://data.europeana.eu/concept/base/128

Then dc:subject will always be present on these contributions, and this error message will never be displayed.